Example/Messaging/Sample/Sample/UserSettings.swift

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import FirebaseMessaging
+
+public final class UserSettings: ObservableObject {
+  @Published public var isAutoInitEnabled: Bool = Messaging.messaging().isAutoInitEnabled {
```

Comment:
This may not work super well depending on what time it's initialized - potentially injecting Messaging to get the initial value? 

It also won't be in sync with `Messaging.messaging().isAutoInitEnabled` if it gets modified separately (I realize this likely won't happen, but it's something that could happen). I think it's fine for now given this is a sample but potentially refactor afterwards!

Reply:
Sounds good. I will keep in mind and observe initialization would cause any issues.